### PR TITLE
Allow to set PoO for others

### LINKED ISSUE #3314: Allow to set point of orders for other users
**Currently:**
Currently a participant is only allowed to add a point of order for themselves

**Reproduction:**
1. activate PoOs
2. Navigate to a LoS-detail-view
3. add a PoO for yourself
4. add a different participant 
5. you cannot add PoO as the type for the other particiapnt

**Wanted:**
It should be possible to add a PoO for another participant (similar to how you can add the type conra/pro etc. for a participant)

This possibility should be activatable via a setting field in the LoS-Settings

### PR DESCRIPTION
**Pull request by Elblinator**
closes #3314 

needs:
https://github.com/OpenSlides/openslides-backend/pull/2231#pullrequestreview-1893426419  and https://github.com/OpenSlides/openslides-meta/pull/33/files and https://github.com/OpenSlides/openslides-autoupdate-service/pull/851

### CODE REVIEW

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
 
     private _listOfSpeakers: ViewListOfSpeakers | null = null;
 
-    private pointOfOrderEnabled = false;
+    public pointOfOrderEnabled = false;
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:156]: Does not seem like this needs to be public

File: client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts [hunk 2]
```diff
@@ -363,6 +363,12 @@ export const meetingSettings: SettingsGroup[] = fillInSettingsDefaults([
                         label: _(`Enable interposed questions`),
                         type: `boolean`
                     },
+                    {
+                        key: `list_of_speakers_can_create_point_of_order_for_others`,
+                        label: _(`Enable point of orders for other participants`),
+                        helpText: _(`Requires permission can manage list of speaker`),
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts:369]: either use the technical term `list_of_speakers.can_manage` (probably not, though) or form a full sentence like "Requires permission to manage lists of speakers"

**Review by jsangmeister — CHANGES_REQUESTED** [review]
You should also update the meta repo & the default setting values

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/modules/point-of-order-dialog/components/point-of-order-dialog/point-of-order-dialog.component.ts [hunk 3]
```diff
@@ -50,40 +81,65 @@ export class PointOfOrderDialogComponent {
...
-            this.meetingSettings.get(`list_of_speakers_enable_point_of_order_categories`),
-            this.categoriesSubject
-        ]).subscribe(([enabled, categories]) => {
-            const show = categories.length && enabled;
-            const categoryForm = this.editForm.get(`category`);
-            if (show) {
-                categoryForm.setValidators([Validators.required]);
-                if (!categories.map(cat => cat.id).includes(categoryForm.value)) {
-                    categoryForm.setValue(categories[0].id);
+        this.operator.userObservable.subscribe(user => (this._currentUser = user)),
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/modules/point-of-order-dialog/components/point-of-order-dialog/point-of-order-dialog.component.ts:88]:
```suggestion
        this.operator.userObservable.subscribe(user => (this._currentUser = user));
```
or am I missing something here?

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Review by rrenkert — CHANGES_REQUESTED** [review]
The user shown in the LoS is always the currently logged in user, not the the selected one.

**Review by rrenkert — APPROVED** [review]

**Pull request merged**